src/components/nav/Nav.jsx

```diff
@@ -37,6 +47,16 @@ const Nav = () => {
           </li>
         ))}
       </ul>
+
+      <ul className="mobile-navUl" style={{ transform: openMenu ? 'translateX(0)' : 'translateX(100%)' }}>
```

Comment:
Please consider creating a dedicated class and adding it conditionally (as you did with the styles) that also makes the element invisible (i.e. `display: none`). Currently, it does the job of moving the nav to the side, however, it is overflowing the viewport width.

Bear in mind that you might need an animation for this to work since `display` is not animatable.

![image](https://user-images.githubusercontent.com/26911718/173876773-3e7f89dc-f4f2-47d4-96f5-ceaa6da0ef05.png)

Reply:
hey @rskinnerc , thanks for the review 😄 
I think once we create the mobile version on the pages, the overflowing issue will be fixed, right now it's happening because the content of the page is overflowing.